views/partials/feedback.hbs

```diff
@@ -0,0 +1,27 @@
+{{#if showFeedback}}
+  {{#if debugging}}
+  <script>document.cookie = 'smcx_0_last_shown_at=;expires=Thu, 01 Jan 1970 00:00:01 GMT;';</script>
+  {{/if}}
+  <style>
+/* By default min height is 300 and position
+     in the center (and can't be tweaked).
+     Also we want only screens with enough
+     space to show the pop up.
+
+     TODO: Might be interesting to animate the
+     entrance instead of just popping up
+  */
+.smcx-widget.smcx-modal {
+  height: 220px !important;
+  bottom: 0 !important;
+  right: 0 !important;
+  top: auto !important;
+  left: auto !important;
+}
+
+.smcx-modal-content {
```

Comment:
Could avoid `!important` for this selector by adding more specificity to CSS selectors.